### DISCUSSION TITLE
Updates?

### DISCUSSION THREAD

**Discussion by enigma-128** [trunkmaster/nextspace#375] [Q&A]
Does NEXTSPACE have the ability to update itself? Also, is there a forum of sorts for NEXTSPACE/GNUstep?
Sorry if this is the wrong place to submit this question, just don't know where else to put it.

**Comment by trunkmaster** [comment]
> Does NEXTSPACE have the ability to update itself? 

What do you mean "update itself"? NEXTSPACE releases distributed as RPM packages. Update can be made with package manager.

> Also, is there a forum of sorts for NEXTSPACE/GNUstep?

Only this github pages is available.

**Comment by Letterus** [comment]
There is #gnustep at IRC Freenode - http://freenode.net/
And someone opened up #nextspace as well. ;-)

With TalkSoup there is a GNUstep based IRC client (which needs some love as most GNUstep apps…): http://gap.nongnu.org/talksoup/
